packages/lb-annotation/src/core/pointCloud/store/index.ts

```diff
@@ -719,6 +722,20 @@ class PointCloudStore {
   public setAttribute(attribute: string) {
     this.currentAttribute = attribute;
   }
+
+  public setSubAttribute(key: string, value: string) {
+    if (!this.cacheSegData) {
+      return;
+    }
+    this.cacheSegData.subAttribute = {
+      ...this.cacheSegData.subAttribute,
+      [key]: value,
```

Comment:
Need to distinguish between incremental additions and full-scale changes.

Reply:
Can only be added or replaced without deletion, so there is no need to distinguish